src/rust/include/rust/orchard.h

```diff
@@ -101,16 +86,24 @@ OrchardBatchValidatorPtr* orchard_batch_validation_init();
 void orchard_batch_validation_free(OrchardBatchValidatorPtr* batch);
 
 /// Adds an Orchard bundle to this batch.
-///
-/// Currently, only RedPallas signatures are batch-validated.
 void orchard_batch_add_bundle(
     OrchardBatchValidatorPtr* batch,
     const OrchardBundlePtr* bundle,
-    const unsigned char* txid);
+    const unsigned char* sighash);
 
 /// Validates this batch.
 ///
 /// Returns false if any item in the batch is invalid.
+///
+/// ## Consensus rules
+///
+/// [§4.6](https://zips.z.cash/protocol/protocol.pdf#actiondesc):
+/// - Canonical element encodings are enforced by `orchard_bundle_parse`.
+/// - SpendAuthSig^Orchard validity is enforced here.
+/// - Proof validity is enforced here.
+///
+/// [§7.1](https://zips.z.cash/protocol/protocol.pdf#txnencodingandconsensus):
+/// - `bindingSigOrchard` validity is enforced here.
```

Comment:
If I understand correctly, all of the transaction-specific consensus rules except for type and canonical encoding rules are now enforced here, so we should say that in the comment.